### ISSUE TITLE
"Record id is not found in record" after create a new class

### ISSUE THREAD

**Issue by brunoccr** [celador/ForceCode#349]
**Steps to reproduce:**

1. Logged in org
2. Create New Class
3. Save to deploy first time
4. Change something in class and save again

![image](https://user-images.githubusercontent.com/5890352/71678771-bbdca300-2d64-11ea-9ab8-f58d5faad91c.png)

**Workaround:**

Restart VS Code and save class file to compile again.

**Comment by brunoccr** [comment]
In containerService.ts:

```js
public createContainer() { 
    var self: Container = this;
    return vscode.window.forceCode.conn.tooling
      .sobject('MetadataContainer')
      .create({ name: 'ForceCode-' + Date.now() })
      .then(res => {
        self.containerId = res.id;
      })
      .then(() => {
        return vscode.window.forceCode.conn.tooling
          .sobject(self.toolingType)
          .find({ Name: self.name, NamespacePrefix: vscode.window.forceCode.config.prefix || '' })
          .execute()
          .then((records: any) => {
            self.records = records; /* THIS NOT RETURN ANY RECORDS */
            return records;
          });
      });
  }
```

**Comment by daishi4u** [comment]
This is fixed in 3.21.4

**Comment by ctinghino** [comment]
I'm experiencing this problem with ForceCode 3.22.2 on a visualforce page.
